doc/admin-api-cn.md

```diff
@@ -67,5 +68,35 @@ Date: Sat, 31 Aug 2019 01:17:15 GMT
 
 目前是直接返回与 etcd 交互后的结果。
 
+### 运算符列表
+
+|运算符   |描述       |例子|
+|--------|-----------|-------|
+|==      |相等      |{"arg_name", "==", "json"}|
```

Comment:
only for string compare?

Reply:
Yes, it is a string object if we get value from 'ngx.var'.